index.html

```diff
@@ -561,35 +581,113 @@ <h4>Script Type</h4>
           </pre>
         </section>
         <section>
-          <h4>Primary Language</h4>
-          <p>The <dfn>Primary Language</dfn> is a mandatory property of a <a>DAPT Script</a>
+          <h4>Default Language</h4>
+          <p>The <dfn>Default Language</dfn> is a mandatory property of a <a>DAPT Script</a>
             which represents the default language for the <a>Text</a> content of <a>Script Events</a>.
-            This language may be the <a>Original</a> language or a <a>Translation</a> language.
+            This language may be the <a>Original Language</a> or a <a>Translation</a> language.
             When it represents a <a>Translation</a> language, it may be the final language
             for which a dubbing or audio description <a>script</a> is being prepared,
             called the <dfn>Target Recording Language</dfn> or it may be an intermediate, or pivot, language
-             used in the workflow.
+            used in the workflow.
           </p>
 
-            <p>The <a>Primary Language</a> is represented in a <a>DAPT Document</a> by the following structure and constraints:
+          <p>The <a>Default Language</a> is represented in a <a>DAPT Document</a> by the following structure and constraints:
             <ul>
               <li>the <code>xml:lang</code> attribute MUST be present on the <code>&lt;tt&gt;</code> element and its value MUST NOT be empty.</li>
             </ul>
           </p>
           <p class="note">All text content in a <a>DAPT Script</a> has a specified language.
```

Comment:
Should it be "All text content ... have a specified language"?

Reply:
I think "content" is singular not plural.